styles/App.css

```diff
@@ -487,6 +487,7 @@ li.quiz-answer .answer-text {
     box-shadow: 2px 2px 5px black;
     border-radius: 7vh;
     transition: filter 0.1s ease-in-out;
+    text-transform: capitalize;
```

Comment:
I don't think it's the best place to capitalize letters. It would be better to do it in a place responsible for generating questions -> i.e. questionGenerator

Reply:
Corrected